final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +110,78 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  'name' => 'Swamp Rat',
+  'smell' => '10 day past due milk carton',
+  'weight' => 200,
+  'citiesDestroyed' => ["Seattle", "Spokane", "Tacoma"],
+  'luckyNumbers' => ['24','16','18'],
+  'address' => ['number' => '8888', 'street' => 'Swamp Ave', 'State' => 'Washington', 'zip' => '98229']
+}
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+attr_reader :name, :super_power, :age
+def initialize(name, super_power, age)
+  @name = name
+  @super_power = super_power
+  @age = age
+
+  arch_nemesis = 'The Syntax Error'
```

Comment:
Even those these are static values, we would want them to be instance variables of the class - similar to L135-137 above.